CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [0.11.1](https://github.com/stellargraph/stellargraph/tree/v0.11.1)
+
```

Comment:
Maybe we could provide a short blurb here that makes it clear that this is just fixing the anaconda issues?

```suggestion

This bugfix release contains the same code as 0.11.0, and just fixes the metadata in the Anaconda package so that it can be installed successfully.

```

Or fold it into the other text. My concern is that a user wouldn't directly understand the consequence (and how it affects them) of the fix currently described.